src/main/java/pl/wip/wipplatformbe/requests/RegisterRequest.java

```diff
@@ -1,11 +1,33 @@
 package pl.wip.wipplatformbe.requests;
 
-import pl.wip.wipplatformbe.requests.base.AuthenticationRequest;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import pl.wip.wipplatformbe.validation.UniqueEmail;
+import pl.wip.wipplatformbe.validation.UniqueUsername;
 
-public class RegisterRequest extends AuthenticationRequest {
+public class RegisterRequest {
+    @NotBlank(message = "Email is required")
+    @Email(message = "Email should be valid")
+    @UniqueEmail
+    private String email;
+
+    @NotBlank(message = "Username is required")
+    @Size(min = 3, max = 15, message = "Username must be between 3 and 15 characters")
+    @UniqueUsername
     private String username;
 
+    @NotBlank(message = "Password is required")
+    @Size(min = 6, message = "Password must be at least 6 characters long")
+    private String password;
```

Comment:
Could you add validation that password must have at least one lowercase, uppercase, number and special character?